pkg/reconciler/pingsource/resources/mt_receive_adapter.go

```diff
@@ -50,6 +54,16 @@ func MakeMTReceiveAdapter(args MTArgs) *v1.Deployment {
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: system.Namespace(),
 			Name:      args.MTAdapterName,
+			OwnerReferences: []metav1.OwnerReference{
```

Comment:
Setting the owner ref to the controller deployment seems like a bug to me, this means that the dataplane and control plane lifecycles are tied together and an upgrade of the control plane takes down the dataplane.